internal/pkg/midi/event.go

```diff
@@ -135,8 +141,91 @@ func ControlChangeEvent(channel, function, value uint8) Event {
 
 // PitchBendEvent accepts a value in range -1.0 to 1.0
 func PitchBendEvent(channel uint8, val float64) Event {
-	target := int(float64((1<<14)-1) * val)  // valid 14-bit pitch-bend range
-	msb := uint8((target >> 7) & 0b01111111) // filtering bit that is beyond valid pitch-bend range when val>1.0, just in case
-	lsb := uint8(target & 0b01111111)        // filtering out one bit of msb, feels good man
+	target := int(float64((1<<14)-1) * ((val + 1.0) / 2.0)) // valid 14-bit pitch-bend range
+	msb := uint8((target >> 7) & 0b01111111)                // filtering bit that is beyond valid pitch-bend range when val>1.0, just in case
+	lsb := uint8(target & 0b01111111)                       // filtering out one bit of msb, feels good man
 	return Event{PitchWheelChange | channel, lsb, msb}
 }
+
+// todo:
+//   implement system exclusive - System Exclusive (data dump) 2nd byte= Vendor ID followed by more data bytes and ending with EOX
```

Comment:
Just use some existing midi library ffs, but maybe this is not in the scope of this PR

Reply:
That could be a good idea for 1.0 release, thanks for suggesting that